1.8/administration/categories-and-forums.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Categories and Forums Management"
+categories: [administration]
+---
+
+## About Categories and Forums
+
+This is most basic guideline how to make and edit your first forums and categories at your forum.
+
+Categories and forums are root of your forum. It is suggested that they should be in some order so users and visitors can get good view of all categories and forums of your forum. There will be explained basic princip of how categories and forums work and what they are for.
+
+## Categories
+
+In Category you can't write anything, you can't open threads or post. They are made just for organisation. Inside them there are forums and sub-forums wich are here for threads and posting.
+
+
+### So what are Categories for then?
+
+Well, you can't just make some forum / subforum without it's parent. They cannot be alone. They needs parent Category in wich they are put. Categories are mostly general names of forums under it.
+For example, we have **category** named Community and under it **forums** like General Discussion, Showcase etc.
+
+### What Category can contain?
+
+Well, when creating new category they need to contain:
+- **Forum Type** - Forum type should be **category**.
+- **Title** - name of your category. Multiple categories can have same name. It is not it's id.
```

Comment:
I think this should be:

"**Title** - name of your category. Multiple categories can have same name but we don't recommend this as it may become confusing"